docker/Dockerfile.faux

```diff
@@ -10,7 +10,7 @@ RUN $AG update && $AG install openjdk-8-jre
 
 RUN $AG update && $AG install openjdk-8-jdk git
 
-ENV BACHASH=d97c917a5ad24e6c9b45f49a5b01efffda9e820e
+ENV BACHASH=ef4d30bf027c5697356931da4fedb91b6a58462e
```

Comment:
There is also a reference to this in docker/Dockerfile.test_bacnet that needs to be updated.